app/javascript/packs/flagUserModal.jsx

```diff
@@ -160,7 +160,7 @@ export function FlagUserModal({ modCenterArticleUrl, authorId }) {
                 destructive
                 variant="primary"
                 className="mr-2"
-                id="confirm-flag-user-action"
+                id="confirm-flag-user-action-inactive"
```

Comment:
I added `-inactive` so that the current FlagUser flow in the ModPanel isn't confused 😅